src/database/migrations/20200311060335-create-acommodation-likes-and-unlikes.js

```diff
@@ -0,0 +1,32 @@
+export default {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('AcommodationLikesAndUnlikes', {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.STRING
+      },
+      userId: {
+        type: Sequelize.STRING
+      },
+      name: {
+        type: Sequelize.STRING
+      },
+      imageUrl: {
+        type: Sequelize.STRING
+      },
```

Comment:
I don't think this is necessary to week
I would recommend the use of relationships to get the details of a user who made alike 
and just keep his/her id

Reply:
This will be helpful on the front-end

Reply:
with the use of relationship you can make sure that you return them both
instead of keeping the some data twice

Reply:
Feedback Attend to. Thanks, @blaise82